discoverx/dx.py

```diff
@@ -117,6 +117,7 @@ def scan(
         rules="*",
         sample_size=10000,
         what_if: bool = False,
+        data_source_formats: list[str] = ["DELTA"],
```

Comment:
Could you please document this in the docstring along with the possible values?

Reply:
Done